server/database/users_test.js

```diff
@@ -0,0 +1,15 @@
+const pool = require('./pool').pool
+
+const clearTestUserTable = (err, request, response) => {
```

Comment:
This function is doing two things:
1. Handling errors from the calling context
2. Running a database query

This creates a coupling between this function and the context in which it is called: i.e. it assumes that the clearTestUserTable will be called in some async process.

Reply:
Thanks, I'll amend this